mi/dataset/parser/test/test_phsen_abcdef_dcl.py

```diff
@@ -210,6 +170,11 @@ def test_bad_checksum_recovered(self):
         in_file = self.open_file(input_file)
         parser = self.create_rec_parser(in_file)
         particles = parser.get_records(total_records)
+
```

Comment:
should this remain as part of the test?

functions starting with `test` will automatically get run by nose. So, if you need to regenerate test yaml files, you can put them in a function called `generate_test_output` or something similar.